ha/factory.go

```diff
@@ -0,0 +1,51 @@
+package ha
+
+import (
+	"fmt"
+	"github.com/cashapp/blip"
+	"sync"
+)
+
+type HAManagerFactory interface {
```

Comment:
Remove "HA" prefix. In Go, pkg name is considered part of the full name context, so `ha.ManagerFactory` is the idiomatic style.